src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    height: 16px;
+    width: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  @for $value from 0 through 5 {
+    &--#{$value} :nth-child(-n + #{$value}) {
```

Comment:
There is an extra space before the pseudo-class ':nth-child'. It should be '&--#{$value}:nth-child(-n + #{$value})' without the space.